public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -119,6 +120,16 @@ class UnConnectedExploreToolbar extends PureComponent<Props> {
       contextSrv.hasAccess(AccessControlAction.DashboardsCreate, contextSrv.isEditor) ||
       contextSrv.hasAccess(AccessControlAction.DashboardsWrite, contextSrv.isEditor);
 
+    const isLargerExploreId = largerExploreId === exploreId;
```

Comment:
I know we do it in many places, but I think `ExploreToolbar` shouldn't know if it's larger than the other pane (ideally shouldn't need to know `exploreId` at all). Could we instead pass `mode` down to the toolbar and decide what interactions are available based on the `mode`?

Reply:
Just to be sure I understand you, your suggestion is instead of getting a `largerExploreId`, it would get something like `largerExploreMode` set as a boolean of true or false?

Reply:
I was thinking about an `enum SizeMode { NORMAL, MAXIMIZED, MINIMIZED }`? To be honest, it's a really minor thing about readability - I think it'd be just more explicit and natural. To be it feels like `ExploreToolbar` shouldn't be aware of which pane is larger, and instead, it should know if it was maximized or not by the parent container/split view.

Reply:
One of the comments from the feedback session involved changing the +/- buttons to < and > , and which one to display depends on the explore ID and size mode. I think we might not be able to avoid this, but I also think the ExploreToolbar knowing what ExploreID it has is fine.